### DISCUSSION TITLE
Oqtane 4.0 Released

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#2933] [General]
Release Announcement: https://www.oqtane.org/blog/!/63/announcing-oqtane-4-0-for-net-7

CSS & JavaScript: https://www.oqtane.org/blog/!/61/css-js-resources

And Routable Modules: https://www.oqtane.org/blog/!/62/routable-modules

**Comment by sbwalker** [comment]
I am thinking that a virtual presentation may be the best way to showcase the features in Oqtane 4.0.

**Reply by vnetonline** [reply]
Happy to help let me know how I can

**Reply by vnetonline** [reply]
I want to work on documentation but I need to understand how everything hangs together … which I was involved from Version 1.0

**Comment by sbwalker** [comment]
One item I should mention as part of the 4.0 release is the change in positioning and messaging for the framework. I actually published a post about this on LinkedIn almost a year ago: https://www.linkedin.com/posts/shaunbrucewalker_cms-blazor-dxp-activity-6958408190665363456-sOvu - and I started including the https://blazorcms.net domain name in release announcements since then. 

Basically I came to the realization that there are 2 distinct definitions for "CMS" - one for Users and one for Developers.  When I created Oqtane I was resisting the "CMS" classification because I was only thinking about the User perspective - and I could not see the value in creating yet another CMS which helps users manage content (as there are already so many powerful and mature products which satisfy this need in the market). 

However, I completely missed the fact that Developers like to use the "CMS" classification to describe web platforms which are dynamic and can be easily extended with custom functionality. Obviously there are a lot of existing products which fit this classification as well, however Developers want to use their preferred technology stack, language, and UI frameworks to build software. So in this sense, there is a need for a "Blazor CMS" which enables Developers to use the Blazor development model to build dynamic applications. 

So after many conversations with Developers over the past 5 years I have finally decided to stop resisting the "CMS" classification - and embrace it instead. This reminds me of the old English proverb "If it walks like a duck, and quacks like a duck, then it is a duck". I am drawing attention to this now to provide clarity to the Oqtane community that the positioning and messaging  has changed (ie. "CMS" is no longer a bad word). Oqtane is a CMS and Application Framework for Blazor and .NET MAUI (targeted at Developers). You will hear more about this in the upcoming .NET Rocks! show which will be published this Thursday. 

You will also notice the use of another domain name in release announcements: https://blazorkit.net - from an application framework perspective it is more descriptive than Oqtane and makes it clear what the underlying technology platform is based on. Note that I do not plan to change the actual Oqtane brand name at this point - but these alternate domain names can be used in conversations with clients and in marketing channels to promote the framework.

**Comment by sbwalker** [comment]
I would also like to kick off a "Core Team" monthly meeting fairly soon. Initially I am thinking that I will invite anyone who has contributed a pull request to Oqtane to attend - as a pull request represents a genuine interest and commitment from an individual to utilize the framework. This is over 40 people at this point so it will be a rather large group (assuming that most are willing/able to join). The goal of the monthly meeting will be to gather feedback and share information on the development progress and milestones.

**Reply by vnetonline** [reply]
@sbwalker I am very much interested in being part of the Core Team Count me in !!!

**Reply by vnetonline** [reply]
> I would also like to kick off a "Core Team" monthly meeting fairly soon. Initially I am thinking that I will invite anyone who has contributed a pull request to Oqtane to attend - as a pull request represents a genuine interest and commitment from an individual to utilize the framework. This is over 40 people at this point so it will be a rather large group (assuming that most are willing/able to join). The goal of the monthly meeting will be to gather feedback and share information on the development progress and milestones. 

@sbwalker Have these monthly meetings started yet

**Reply by leigh-pointer** [reply]
@sbwalker no! But I think this is something that Oqtane should manage!

**Comment by sbwalker** [comment]
There was a nice mention of Oqtane in the ASP.NET Community Standup last month - https://youtu.be/MaLjiR9YSbs?t=476

**Reply by vnetonline** [reply]
Great to see Oqtane is getting traction

**Comment by sbwalker** [comment]
.NET Rocks! episode here: https://www.linkedin.com/feed/update/urn:li:activity:7080167039880814592/

**Reply by leigh-pointer** [reply]
@sbwalker just tuned in to it this morning. I enjoyed it, very interesting.

**Reply by vnetonline** [reply]
Yes, I agree, very interesting and informative I also enjoyed it!
